### PR TITLE
DED-258 adding sample apps links to auth documentation

### PR DESCRIPTION
**Pull request by kamilatokarska**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira]()

# Description

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/authorization/index.mdx [hunk 1]
```diff
@@ -44,6 +44,13 @@ If you'd like to see the authorization flows explained live, watch our video tut
   allowfullscreen
 ></iframe>
 
+# Sample Authorization Apps
+
+We've prepared two apps which include fully implemented authorization code samples. You can test them, experiment with them, or modify one of this app’s code and extend its functionality.
```
Comment (ale-kacperczyk) [inline src/pages/authorization/index.mdx:49]:
```suggestion
We've prepared two apps with fully implemented authorization flows. You can test them, experiment with them, or modify their code and extend their functionality.
```

File: src/pages/authorization/index.mdx [hunk 2]
```diff
@@ -44,6 +44,13 @@ If you'd like to see the authorization flows explained live, watch our video tut
...
+
```
Comment (ale-kacperczyk) [inline src/pages/authorization/index.mdx:50]: - <a href="https://github.com/livechat/sample-app-popup-auth" target="_blank">Sample app with the popup flow</a>
- <a href="https://github.com/livechat/sample-app-redirect-auth" target="_blank">Sample app with the redirect flow</a>

Comment (ale-kacperczyk) [reply src/pages/authorization/index.mdx:50]: Or maybe we could use the `<SectionLink>` component here? 👆🏻

File: src/pages/authorization/index.mdx [hunk 3]
```diff
@@ -44,6 +44,13 @@ If you'd like to see the authorization flows explained live, watch our video tut
...
+Currently, two flows for obtaining an access token from the [LiveChat Accounts SDK](/authorization/sign-in-with-livechat#accounts-sdk) are available: <a href="https://github.com/livechat/sample-app-popup-auth" target="_blank">popup flow</a> and <a href="https://github.com/livechat/sample-app-redirect-auth" target="_blank">redirect flow</a>. Please check out our Sample authorization app tutorial if you want more information about our sample apps.
```
Comment (ale-kacperczyk) [inline src/pages/authorization/index.mdx:51]:
```suggestion
The apps demonstrate how to implement two flows from the [LiveChat Accounts SDK](/authorization/sign-in-with-livechat#accounts-sdk): the [popup flow](/authorization/sign-in-with-livechat#popup-1) and the [redirect flow](/authorization/sign-in-with-livechat#redirect-1).

Please check out our Sample authorization app tutorial if you want more information about our sample apps.
```

**Review by ale-kacperczyk — COMMENTED** [review]
A couple suggestions

File: src/pages/authorization/index.mdx (hunk 2 above)
Comment (kamilatokarska) [reply src/pages/authorization/index.mdx:50]: Tomorrow, I will see how it looks and decide:) However, it seems like a very good idea

File: src/pages/authorization/index.mdx [hunk 4]
```diff
@@ -44,6 +44,15 @@ If you'd like to see the authorization flows explained live, watch our video tut
   allowfullscreen
 ></iframe>
 
+# Sample Authorization Apps
+
+We've prepared two apps with fully implemented authorization flows. You can test them, experiment with them, or modify their code and extend their functionality.
+
+The apps demonstrate how to implement two flows from the [LiveChat Accounts SDK](/authorization/sign-in-with-livechat#accounts-sdk): the [popup flow](/authorization/sign-in-with-livechat#popup-1) and the [redirect flow](/authorization/sign-in-with-livechat#redirect-1).
+
+Please check out our Sample authorization app tutorial if you want more information about our sample apps.
```
Comment (kamilatokarska) [inline src/pages/authorization/index.mdx:57]: Do we already know what the link here will be?

File: src/pages/authorization/index.mdx (hunk 2 above)
Comment (kamilatokarska) [reply src/pages/authorization/index.mdx:50]: I have added normal links, as the `<SecrionLink>' component was not working when I tried. However, it still looks very similar to the `<SectionLink>' :)

File: src/pages/authorization/index.mdx (hunk 4 above)
Comment (ale-kacperczyk) [reply src/pages/authorization/index.mdx:57]: https://developers.livechat.com/updates/learn-authorization-with-sample-apps

File: src/pages/authorization/index.mdx [hunk 5]
```diff
@@ -44,6 +44,19 @@ If you'd like to see the authorization flows explained live, watch our video tut
   allowfullscreen
 ></iframe>
 
+# Sample Authorization Apps
```
Comment (ale-kacperczyk) [inline src/pages/authorization/index.mdx:47]:
```suggestion
# Sample authorization apps
```

Comment (ale-kacperczyk) [reply src/pages/authorization/index.mdx:47]: other headlines in this doc are sentence case

File: src/pages/authorization/index.mdx (hunk 2 above)
Comment (ale-kacperczyk) [reply src/pages/authorization/index.mdx:50]: ok

**Review by ale-kacperczyk — APPROVED** [review]

**Comment by ale-kacperczyk** [comment]
I'm sorry to re-open this PR, but I just realized we should also link these sample apps in:
https://developers.livechat.com/docs/authorization/sign-in-with-livechat#popup-1
and
https://developers.livechat.com/docs/authorization/sign-in-with-livechat#redirect-1

File: src/pages/authorization/index.mdx [hunk 6]
```diff
@@ -44,6 +44,19 @@ If you'd like to see the authorization flows explained live, watch our video tut
...
+
+We've prepared two apps with fully implemented authorization flows. You can test them, experiment with them, or modify their code and extend their functionality:
+
+- <a href="https://github.com/livechat/sample-app-popup-auth" target="_blank">Sample app with the popup flow</a>
+
+- <a href="https://github.com/livechat/sample-app-redirect-auth" target="_blank">Sample app with the redirect flow</a>
+
+The apps demonstrate how to implement two flows from the [LiveChat Accounts SDK](/authorization/sign-in-with-livechat#accounts-sdk): the [popup flow](/authorization/sign-in-with-livechat#popup-1) and the [redirect flow](/authorization/sign-in-with-livechat#redirect-1).
+
+Please check out our <a href="https://developers.livechat.com/updates/learn-authorization-with-sample-apps" target="_blank">Sample authorization app tutorial</a> if you want more information about our sample apps.
```
Comment (ale-kacperczyk) [inline src/pages/authorization/index.mdx:57]:
```suggestion
To learn more about the sample apps and implementing Accounts SDK, read our <a href="https://developers.livechat.com/updates/learn-authorization-with-sample-apps" target="_blank">blog post</a>.
```

**Pull request merged**